### PR TITLE
Removed html from examples.md and added references to api, _sidebar and README .md files.

### PR DESCRIPTION
**Pull request by lazarovziv**
added backpressure, https, middleware and broadcast examples from the examples folder.

### CODE REVIEW

**Comment by cirospaciari** [comment]
Great! but we need some more tweaks.
Can you remove # socketify.py section? change the ## Examples to # Examples and also add the link to the summary here https://github.com/cirospaciari/socketify.py/blob/main/docs/README.md after API Reference and here https://github.com/cirospaciari/socketify.py/blob/main/docs/_sidebar.md. In API Reference page we need a link to examples too at the end, you can see how its created in the other docs source.

**Comment by lazarovziv** [comment]
You mean remove the html stuff?
And where to put the links? Below the # examples?

**Comment by cirospaciari** [comment]
> Reference

Yeah no need of this section:
https://github.com/cirospaciari/socketify.py/pull/121/files#diff-42c9e8de9b02b38190aa003b14db6be144d9d6f34b266cf401330e9e62a3bba9R1-R22

File: docs/api.md [hunk 1]
```diff
@@ -229,4 +229,7 @@ class MiddlewareRouter:
     def connect(self, path, handler):
     def trace(self, path, handler):
     def any(self, path, handler):
-```
\ No newline at end of file
+```
+
+# Examples
+- [Examples](examples.md)
```
Comment (cirospaciari) [inline docs/api.md:235]: Here you can do 
```
### Next [Examples(examples.md)
```

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Examples
+
+Middleware
```
Comment (cirospaciari) [inline docs/examples.md:3]: We should give more complete description but I can add it later

**Review by cirospaciari — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 1 above)
Comment (lazarovziv) [reply docs/api.md:235]: do you mean:
# Examples
### Next [Examples](examples.md)
?

Comment (lazarovziv) [reply docs/api.md]: done.

**Review by cirospaciari — APPROVED** [review]

**Pull request merged**